dev-util/bitcoin-tx/bitcoin-tx-0.15.1.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 2010-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools bash-completion-r1
+
+MyPV="${PV/_/}"
+MyPN="bitcoin"
+MyP="${MyPN}-${MyPV}"
+BITCOINCORE_COMMITHASH="7b57bc998f334775b50ebc8ca5e78ca728db4c58"
+KNOTS_PV="${PV}.knots20171111"
+KNOTS_P="${MyPN}-${KNOTS_PV}"
+
+IUSE="knots libressl"
+
+DESCRIPTION="Command-line Bitcoin transaction tool"
+HOMEPAGE="http://bitcoincore.org/ http://bitcoinknots.org/"
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~amd64-linux ~arm ~arm64 ~mips ~ppc ~x86 ~x86-linux"
+
+SRC_URI="
+	https://github.com/${MyPN}/${MyPN}/archive/${BITCOINCORE_COMMITHASH}.tar.gz -> ${MyPN}-v${PV}.tar.gz
+	http://bitcoinknots.org/files/0.15.x/${KNOTS_PV}/${KNOTS_P}.patches.txz -> ${KNOTS_P}.patches.tar.xz
+"
+CORE_DESC="https://bitcoincore.org/en/2017/11/11/release-${PV}/"
+KNOTS_DESC="http://bitcoinknots.org/files/0.15.x/${KNOTS_PV}/${KNOTS_P}.desc.html"
+
+RDEPEND="
+	!libressl? ( dev-libs/openssl:0[-bindist] ) libressl? ( dev-libs/libressl )
```

Comment:
on separate lines pls, and both `openssl` and `libressl` require `:0=`

Reply:
Here is the link to the on-dep-per line 
https://devmanual.gentoo.org/general-concepts/dependencies/